packages/server/src/channels/telegram/channel.ts

```diff
@@ -25,16 +26,26 @@ export class TelegramChannel extends Channel<TelegramConduit> {
 
   async setupRoutes() {
     this.router.use(
-      '/',
+      '/:token',
       this.asyncMiddleware(async (req, res) => {
         // This is done to make forwarding work
-        req.url = '/'
+        req.url = `/${req.params.token}`
 
         const conduit = res.locals.conduit as TelegramConduit
-        conduit.callback(req, res)
+        if (this.validate(conduit, req)) {
+          conduit.callback(req, res)
+        } else {
+          this.logger.error("Request validation failed. Request probably didn't come from Telegram.")
+
+          res.status(401).send('Auth token invalid')
```

Comment:
Seems like a lot of text that says the same thing. I would remove the logger.error from here and just do
```ts
return res.sendStatus(401)
```

Reply:
Good point!